Kernel/VirtIO/VirtIOConsole.cpp

```diff
@@ -9,6 +9,23 @@
 
 namespace Kernel {
 
+VirtIOConsole::VMRingBuf::VMRingBuf(const StringView region_name, size_t capacity)
+    : m_region{MM.allocate_contiguous_kernel_region(capacity, region_name, Region::Access::Read | Region::Access::Write)}, capacity_in_bytes{capacity} {}
```

Comment:
```suggestion
    : m_region(MM.allocate_contiguous_kernel_region(capacity, region_name, Region::Access::Read | Region::Access::Write)), capacity_in_bytes(capacity) {}
```
To follow the style of the rest of the project